File: programmers/level1/037.py
# 37. 비밀지도
def binary(num,n):
    binary = ''
    while num>0:
        binary= str(num%2) + binary
        num//=2
    binary=binary.rjust(n,'0')
    return binary

def solution(n, arr1, arr2):
    answer = []
    code =''
    for a,b in zip(arr1,arr2):
        code=bin(a|b)[2:].rjust(n,'0')
        code=code.replace('1','#')
        code=code.replace('0',' ')

        answer.append(code)
    return answer

File: programmers/level0/037.java
// 37. 옷가게 할인 받기
class Solution {
    public int solution(int price) {

        if( price < 300000 && price >= 100000){
                price *= 0.95;    
        }else if(price <500000 && price >= 300000){
                price*=0.9;
        }else if ( price >= 500000){
                price*=0.8;
        }
        return price;
    
    }
}